### DISCUSSION TITLE
Not able to run Whisper on Macbook

### DISCUSSION THREAD

**Discussion by nkshtrx** [openai/whisper#1443] [Q&A]
So I'm trying to transcribe through terminal by adding file name but it is showing file not found in directory. How do I solve this? 
<img width="742" alt="Screenshot 2023-06-12 at 4 12 02 PM" src="https://github.com/openai/whisper/assets/135532679/cdcd3452-a53f-47f2-abaa-01fe9ec5f83b">

**Comment by phineas-pta** [comment]
the error message got cut out, could you give the complete message?
